src/Service.Host/client/src/components/PurchaseOrders/CreatePurchaseOrderUt.js

```diff
@@ -137,27 +132,6 @@ function CreatePurchaseOrderUt() {
         }
     }, [order, reduxDispatch]);
 
-    const exchangeRatesItems = useSelector(state =>
```

Comment:
is all this exchange rate stuff ever used on this form? Didn't look like it to me